tests/fixtures/nodejs_mono_repo_without_version/packages/no_version/package.json

```diff
@@ -4,7 +4,7 @@
   "license": "Apache-2.0",
   "repository": "googleapis/repo-automation-bots",
   "engines": {
```

Comment:
I think we should keep the tests and fixtures as is, these are unnecessary changes.

Reply:
This goes for all changes under the tests/ directory, it bloats the PR.